process_manager/views/partials.py

```diff
@@ -94,7 +97,35 @@ def process_table(request: HttpRequest) -> HttpResponse:
     )
 
 
+def handle_errors(
+    view_func: Callable[[HttpRequest], HttpResponse],
+) -> Callable[[HttpRequest], HttpResponse]:
+    """Decorator to handle errors.
+
+    Args:
+        view_func: The view function to be wrapped.
+
+    Returns:
+        The wrapped view function.
+    """
+    logger = logging.getLogger("django")
+
+    def wrapped_view(request, *args, **kwargs) -> HttpResponse:  # type: ignore
+        try:
+            return view_func(request, *args, **kwargs)
+        except Exception as e:
+            logger.exception(f"Error in view {view_func.__name__}: {e}")
+            return render(
+                request,
+                "process_manager/partials/error_message.html",
```

Comment:
We're missing this file.

Reply:
Was going to ask what should i write for this

Reply:
Probably just "error refresh page content" in red text will do.

Reply:
Would I have to include the same template tags as the message_items.html

Reply:
No, just a span or something with the text in.

Reply:
So we are not actually providing any useful error message to the user, just that something went wrong, right?

Reply:
Yeah. With the current implementation if even one request fails then updates to the table/message feed stop but there is no sign of this to the user. This is an improvement in that in the case of an error there will be some indication to the user but if it was a one-off then the page will update again removing the error if not they have the chance to refresh or investigate further.